### ISSUE TITLE
[BUG] PlayTimes is being recognized as a Legacy Plugin

### ISSUE THREAD

**Issue by OreoKirby** [CodedRed-Spigot/PlayTimes#63]
## Basic Information

**Minecraft Version:**
- Paper 1.21-47

**PlayTimes Version:**
- 1.6.2b
![Screen Shot 2024-07-08 at 11 10 08 AM](https://github.com/CodedRed-Spigot/PlayTimes/assets/1530615/06bb1024-6ff7-4a48-b6f6-e4909d5ca7a7)

---

## Describe the Bug

**Description:**
- Due to an `api-version` not being set Paper is forcing Initialization of Legacy Material Support.

---

## Screenshots

**Screenshots:**
![Screen Shot 2024-07-08 at 11 03 11 AM](https://github.com/CodedRed-Spigot/PlayTimes/assets/1530615/1530a9c4-2736-4745-9b2f-79f879a6558e)

---

## Additional Context

**Additional Context:**
```
[14:12:33 WARN]: [org.bukkit.craftbukkit.legacy.CraftLegacy] Initializing Legacy Material Support. Unless you have legacy plugins and/or data this is a bug!
[14:12:41 WARN]: Legacy plugin Playtimes v1.6.2b does not specify an api-version.
```

**Comment by Dmitrium12** [comment]
Yes, I am facing the same "problem". It doesn't interfere with the plugin, it's just a little annoying.
